function signature and usage missmatch

When building the `/tutorials/frontend/angular-apollo/app-boilerplate/` project an error occurs in the following template template `Auth0Wrapper`

Steps to reproduce

run `npm start`

log
```
ERROR in src/app/Auth/Auth0Wrapper.template.html:2:55 - error TS2554: Expected 0
 arguments, but got 1.

2 <Login *ngIf="!isAuthenticated" (loginHandler)="login($event)"></Login>
                                                        ~~~~~~

  src/app/Auth/Auth0Wrapper.ts:8:16
    8   templateUrl: './Auth0Wrapper.template.html',
                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component Auth0Wrapper.

```

--- a/tutorials/frontend/angular-apollo/app-boilerplate/src/app/Auth/Auth0Wrapper.template.html
+++ b/tutorials/frontend/angular-apollo/app-boilerplate/src/app/Auth/Auth0Wrapper.template.html
@@ -1,5 +1,5 @@
 <Callback *ngIf="isAuthenticated == 'loading'"></Callback>
-<Login *ngIf="!isAuthenticated" (loginHandler)="login($event)"></Login>
+<Login *ngIf="!isAuthenticated" (loginHandler)="login()"></Login>
 
 <App *ngIf="isAuthenticated === true" (logout)="logout()"></App>
 

